test/react-native/features/metadata.feature

```diff
@@ -8,8 +8,9 @@ Scenario: Setting metadata (JS)
   And the event "metaData.jsdata.some_data" equals "set via config"
   And the event "metaData.jsdata.some_more_data" equals "set via client"
   And the event "metaData.jsdata.even_more_data" equals "set via event"
+  And the payload field "events.0.metaData.jsarraydata.items" is an array with 3 elements
```

Comment:
I'm guessing the bug was that the array wasn't present at all?  I mean, would there be some merit is checking what the 3 values actually are?

Reply:
the bug was that it came through as:

```js
{ '0': 'a', '1': 'b', '2': 'c' }
```

instead of


```js
[ 'a', 'b', 'c' ]
```